six/modules/c++/six.sidd/include/six/sidd/SFA.h

```diff
@@ -205,8 +201,10 @@ struct SFAPolyhedralSurface : public SFASurface
         SFASurface(TYPE_NAME)
     {
     }
-    virtual ~SFAPolyhedralSurface()
+
+    virtual SFAPolyhedralSurface* clone() const
     {
+        return new SFAPolyhedralSurface(*this);
     }
 
     std::vector<mem::ScopedCopyablePtr<SFAPolygon> > patches;
```

Comment:
This (and SFAMultiPolygon) need to still be switched to be a vector of scoped cloneables right?  The vector of scoped copyables of SFAPoint is ok as-is